src/test/java/com/mael/payroll/transactions/AddMonthlyEmployeeTest.java

```diff
@@ -12,10 +15,11 @@
 
     @Before
     public void setUp() {
-        PayrollDBFacade payrollDB      = new PayrollDBFacade();
-        AddMonthlyEmployee addMonthlyE = new AddMonthlyEmployee(payrollDB);
+        PayrollDBFacade payrollDB          = new PayrollDBFacade();
+        AddEmployeeTransaction addMonthlyE = new AddMonthlyEmployee(1, "Squiddo", "FishBowl", 1000.0);
```

Comment:
Same as above. Are the values "FishBowl" and 1000.0 needed at all? If so, they should have matching test.